src/pal/tests/palsuite/c_runtime/pow/test1/test1.cpp

```diff
@@ -106,13 +106,19 @@ int __cdecl main(int argc, char **argv)
         { -2.7182818284590452,     1,                     -2.7182818284590452,        PAL_EPSILON * 10 },     // x: -(e)                       expected: e
         { -2.7182818284590452,     PAL_POSINF,             PAL_POSINF,                0 },                    // x: -(e)
 
+        { -1.0,                    PAL_NEGINF,             1.0,                       PAL_EPSILON * 10 },
```

Comment:
Yes, its probably the case that we can expect an exact result here. I was just being consistent with the existing convention here.

There are plenty of things that could be improved. For example, the PAL layer (as a compatibility test) currently checks if `atan2(0.0, -0.1)` is within `0.0000001` digits of PI, (https://github.com/dotnet/coreclr/blob/master/src/pal/src/configure.cmake#L797), such logic is incredibly broken 😄 

I have been working on a separate PR that fixes all of this up, and it will be corrected when that goes out. This PR, at the very least, will fix the validate function to properly check:
* Exact results
* NaN
* -0.0
* +0.0
* Negative Infinity
* Positive Infinity
* Results that are within 15 digits of the expected value